src/Components/test/testassets/BasicTestApp/FormsTest/TypicalValidationComponent.razor

```diff
@@ -70,6 +70,14 @@
         </InputSelect>
         <span>@string.Join(", ", person.HostileStrings)</span>
     </p>
+    <p class="select-bool-values">
+        T/F: 77 + 33 = 100<br>
+        <InputSelect @bind-Value="person.IsSelectMathStatementTrue">
```

Comment:
Both of these seem like common enough scenario (yes / no style question) that it feels like it's worthwhile making it work.